### ISSUE TITLE
Pretrained weights not available 

### ISSUE THREAD

**Issue by tsbiosky** [SizheAn/MARS#8]
Hi, I tired the pretrained weights but the prediction was very bad , the values for XYZ are in [-600,1000].
So I trained the model from scratch. but still get lower performance 

test MAPE =  324.9508972167969
mae for x is 0.1080006290689424
mae for y is 0.09140279204298475
mae for z is 0.09259880900528002

the visualization results show the model always predict the same pose like this

![1000](https://github.com/SizheAn/MARS/assets/31939883/684921a1-ca49-449c-bd00-1f31489df143)
![2000](https://github.com/SizheAn/MARS/assets/31939883/41f6bcd5-0bfd-48a5-af1c-0075465fd4bc)
![1](https://github.com/SizheAn/MARS/assets/31939883/39782342-c24e-43f6-9746-87e75bb55b64)
![500](https://github.com/SizheAn/MARS/assets/31939883/5d550a62-41f4-46e2-be8c-08e72a46b493)
 I don't how to solve this issues 
would you like to update a pretrained weights ? thanks

**Comment by SizheAn** [comment]
How's the loss? And are you following the exact version number in the installation guide? I suspect that could be keras' version problem. I remember I've faced some similar errors before. 

Which version of python, tensorflow, keras are you using? Also, which graphic card and cuda version? TF2 and keras might have some computing issues with RTX 30 series card with old version of cuda (10.x).  Using Keras 2.3.0 and TF 2.2.0, cuda 10.1 with RTX 2080 everything runs smoothly. I recently switched to pytorch and it has no problem with RTX 30 series either. Due to my limited band width, I haven’t uploaded the code to the github. 

I will try to upload the pytorch model and pretrained weights in this page in the future.

**Comment by tsbiosky** [comment]
@SizheAn , thank you for your quick reply ! 
I'm using Keras 2.4.3  TF 2.2.0  cuda 11.8 with RTX 4070Ti , I can't get the same environment and cuda version due to my hardware .

Here is the loss , it converges around  0.0173
![loss](https://github.com/SizheAn/MARS/assets/31939883/62978b0b-ea9e-48cb-8a1f-4e0365e2765b)

Btw, I also tried to use Pointnet to replace CNN , but still get similar results 
  
test MAPE =  316.0719909667969
mae for x is 0.10843670352503432
mae for y is 0.07936162722608399
mae for z is 0.09297142448554381
